packages/backend/src/open_payments/payment_pointer/key/routes.test.ts

```diff
@@ -7,7 +7,11 @@ import { createTestApp, TestContainer } from '../../../tests/app'
 import { Config } from '../../../config/app'
 import { IocContract } from '@adonisjs/fold'
 import { initIocContainer } from '../../..'
-import { AppServices, PaymentPointerContext } from '../../../app'
+import {
+  AppServices,
+  PaymentPointerContext,
```

Comment:
```suggestion
```
These can all use `PaymentPointerKeysContext`

Reply:
Yeah, I thought so and then forgot to do the change 🤦‍♀️